### ISSUE TITLE
Add support for playlists

### ISSUE THREAD

**Issue by a-v-e-s** [pithos/pithos#596]
Pandora offers playlists for premium subscribers, but Pithos has no means of interfacing with these. I'm a bit of a noob but may steadily get to work on adding this functionality. Would really like to have a Pithos app that can access the dozen or so playlists I have on my account from my [PinePhone](https://www.pine64.org/pinephone/) once it is released.

**Comment by JasonLG1979** [comment]
Pithos is an unofficial client. Pandora does not offer a public API. What we can currently do is based on reverse engineering their private API. (it's a bit of an gray area) Also we use the legacy API that does not have access to a lot of the newer features you see on official clients. The legacy API does not offer a lot of the newer features that the official clients now use. I seriously doubt that you can even access playlists on the legacy API.

There was an effort to try to figure out the new API but it was just @TingPing and I so I just gave up. As far as I'm concerned Pithos is pretty much in maintenance mode. I mean it works for what it does, but unless we get a lot of fresh blood to help us I don't see any big new features being added in the near future.

**Comment by BeaverTek** [comment]
What if those playlists were handled locally instead? Is that possible?

**Comment by JasonLG1979** [comment]
> What if those playlists were handled locally instead? Is that possible?

Not really. Downloading songs is a no go and the urls given out by Pandora expire after like 30 mins. The newer API has methods to get fresh urls but the currently used API does not.

Someone would need to get cracking on the new API.
